fenix/app/src/test/java/org/mozilla/fenix/shopping/ReviewQualityCheckBottomSheetStateFeatureTest.kt

```diff
@@ -40,24 +39,24 @@ class ReviewQualityCheckBottomSheetStateFeatureTest {
         ).joinBlocking()
         store.dispatch(ReviewQualityCheckAction.OptOutCompleted(emptyList())).joinBlocking()
 
-        assertFalse(isInvoked)
+        assertEquals(BottomSheetViewState.HALF_VIEW, updatedState)
     }
 
     @Test
     fun `WHEN store state changes to not opted in from initial state THEN callback is invoked`() {
         val store = ReviewQualityCheckStore(emptyList())
-        var isInvoked = false
+        var updatedState: BottomSheetViewState? = null
         val tested = ReviewQualityCheckBottomSheetStateFeature(
             store = store,
-            onRequestStateExpanded = {
-                isInvoked = true
+            onRequestStateUpdate = {
+                updatedState = it
             },
         )
         assertEquals(ReviewQualityCheckState.Initial, store.state)
 
         tested.start()
         store.dispatch(ReviewQualityCheckAction.OptOutCompleted(emptyList())).joinBlocking()
 
-        assertTrue(isInvoked)
+        assertEquals(BottomSheetViewState.HALF_VIEW, updatedState)
```

Comment:
Should this case lead to `FULL_VIEW`?

Also let's update the test names to reflect the new callback with param.